learning/katas/python/Core Transforms/Map/FlatMap/task.py

```diff
@@ -20,7 +20,12 @@
 
 with beam.Pipeline() as p:
 
+  print("This is a Map:")
```

Comment:
I believe there is a separate Map related kata. (https://github.com/apache/beam/blob/master/learning/katas/python/Core%20Transforms/Map/Map/task.py)

That probably covers Map and we can keep this specific to FlatMap.

Reply:
Yea was thinking it might reinforce the learning at the time but it does seem like fluff in retrospect. Reverted the FlatMap bit. 👍